app/frontend/src/components/form/Step6.vue

```diff
@@ -213,8 +214,31 @@ export default {
     async submit() {
       await this.submitForm();
     },
-    downloadPdf() {
-      alert('coming soon');
+    generatePdf(){
+      ipcService
+        .getIPCContentAsPDF(this.submissionDetails.ipcPlan.ipcPlanId)
+        .then(response => {
+          const blob = new Blob([response.data], {
+            type: 'attachment'
+          });
+          this.createDownload(blob, `${this.submissionDetails.ipcPlan.ipcPlanId}.pdf`);
+        })
+        .catch(() => {
+          /*eslint-disable */
+          console.log('Currently unable to complete this request.');
+          /*eslint-enable */
```

Comment:
console.log('Currently unable to complete this request.'); // eslint-disable-line no-console

Reply:
done